<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectcr</class>
 <widget class="QDialog" name="selectcr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CR Password</string>
  </property>
  <widget class="QComboBox" name="comboBoxSelect">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: black; 
font: 700 10pt &quot;Arial&quot;;

</string>
   </property>
   <item>
    <property name="text">
     <string>BIT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CEI_I</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CEI_II</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CEII_I</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CEII_II</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CEIII_I</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CEIII_II</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CEIV_I</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditPassword">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>297</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditConfirmpassword">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>297</width>
     <height>28</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Confirm password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSignUp">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>297</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 9pt &quot;Arial&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Enter password for CR :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
